inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -42,6 +42,7 @@ class UsedCSS extends Table {
 		20220920 => 'make_status_column_index_instead_queue_name',
 		20221104 => 'add_error_columns',
 		20231010 => 'add_submitted_at_column',
+		20231110 => 'add_is_home',
```

Comment:
You need to update the $version above to have the same version like that `20231110`

Reply:
It's been corrected